docs/readthedocs/models/input/grid/transformer2w.md

```diff
@@ -10,41 +10,129 @@ It is assumed, that node A is the node with higher voltage.
 All impedances and admittances are given with respect to the higher voltage side.
 As obvious, the parameter can be used in T- as in 𝜋-equivalent circuit representations.
 
-| Attribute | Unit | Remarks                                                 |
-|:----------|:-----|:--------------------------------------------------------|
-| id        |      | Human readable identifier                               |
-| rSc       | Ω    | Short circuit resistance                                |
-| xSc       | Ω    | Short circuit reactance                                 |
-| gM        | nS   | No load conductance                                     |
-| bM        | nS   | No load susceptance                                     |
-| sRated    | kVA  | Rated apparent power                                    |
-| vRatedA   | kV   | Rated voltage at higher voltage terminal                |
-| vRatedB   | kV   | Rated voltage at lower voltage terminal                 |
-| dV        | %    | Voltage magnitude increase per tap position             |
-| dPhi      | °    | Voltage angle increase per tap position                 |
-| tapSide   |      | true, if tap changer is installed on lower voltage side |
-| tapNeutr  |      | Neutral tap position                                    |
-| tapMin    |      | Minimum tap position                                    |
-| tapMax    |      | Maximum tap position                                    |
+```{eval-rst}
+.. list-table::
+   :widths: 33 33 33
+   :header-rows: 0
+
+   * - Attribute
+     - Un it
+     - Remarks
+
+   * - id
+     -
+     - Human readable identifier
+
+   * - rSc
+     - Ω
+     - Short circuit resistance
+
+   * - xSc
+     - Ω
+     - Short circuit reactance
+
+   * - gM
+     - nS
+     - No load conductance
+
+   * - bM
+     - nS
+     - No load susceptance
+
+   * - sRated
+     - k VA
+     - Rated apparent power
+
+   * - vRatedA
+     - kV
+     - Rated voltage at higher voltage terminal
+
+   * - vRatedB
+     - kV
+     - Rated voltage at lower voltage terminal
+
+   * - dV
+     - %
+     - Voltage magnitude increase per tap position
+
+   * - dPhi
+     - °
+     - Voltage angle increase per tap position
+
+   * - tapSide
+     -
+     - true, if tap changer is installed on lower voltage
+       side
+
+   * - tapNeutr
+     -
+     - Neutral tap position
+
+   * - tapMin
+     -
+     - Minimum tap position
+
+   * - tapMax
+     -
+     - Maximum tap position
+```
 
 A list with some standard transformer types can be found here: `Standard Two Winding Transformer Types`_
```

Comment:
Here should be a link. See: ``line.md``
There are also two additional broken links below.